docs/advanced/advanced.md

```diff
@@ -372,47 +357,37 @@ There are several ways to run K3s in Docker:
 <Tabs>
 <TabItem value='K3d' default>
 
-[k3d](https://github.com/k3d-io/k3d) is a utility designed to easily run K3s in Docker.
-
-It can be installed via the [brew](https://brew.sh/) utility on MacOS:
-
-```bash
-brew install k3d
-```
-
-</TabItem>
-<TabItem value="Docker Compose">
```

Comment:
Why are we removing the docker compose tab? We still have a valid Docker compose available in the repo.

Reply:
As far as I know it's not maintained, tested, or supported. Running it directly via docker, or via k3d, are both better options than the compose file. I guess we can keep it around for now but I don't like steering people towards something that we don't actually recommend.

Reply:
If we don't want to support it, that's fine, we can remove it from the docs, but I think we should potentially remove it from master as well.

Reply:
I'll open a PR to k3s to drop the docker-compose file.